### PR TITLE
fixes #7221 - do not display areas of the org/loc UI unless view rbac

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: app/views/taxonomies/_form.html.erb [hunk 1]
```diff
@@ -12,142 +12,194 @@
...
+          <li><a href="#subnets" data-toggle="tab"><%= _("Subnets") %></a></li>
+        <% end %>
+        <% if User.current.allowed_to?(:view_compute_resources) %>
+          <li><a href="#compute_resources" data-toggle="tab"><%= _("Compute Resources") %></a></li>
+        <% end %>
+        <% if User.current.allowed_to?(:view_media) %>
+          <li><a href="#media" data-toggle="tab"><%= _("Media") %></a></li>
+        <% end %>
+        <% if User.current.allowed_to?(:view_template) %>
+          <li><a href="#template" data-toggle="tab"><%= _("Templates") %></a></li>
```
Comment (thomasmckay) [inline app/views/taxonomies/_form.html.erb]: @isratrade @domcleal Is 'template' supposed to be singular here or plural? It is singular in original code but wished to confirm as it is the only one.

**Comment by thomasmckay** [comment]
@isratrade @domcleal  I could use an example for how to only show the instances of resources that a user has permission to see as well. If the user may only view media "CentOS mirror" then that should be the only choice allowed for addition to the organization.

Note that this will also expose the issue that the form submission controller will then incorrectly end up making the org have _only_ "Centos mirror" since it submits all of the IDs on 'submit' instead of ajax add/remove per list click.

This UI is really not designed to be used in the context of RBAC. Suggestions welcome.

**Comment by thomasmckay** [comment]
... I'd like to keep this pull-request just for not displaying the parts of the UI without view permission. The filtering of items displayed is an entirely separate issue worthy of its own redmine.

File: app/views/taxonomies/_form.html.erb (hunk 1 above)
Comment (isratrade) [inline app/views/taxonomies/_form.html.erb]: @thomasmckay, yes, it should be `:view_templates`

**Comment by isratrade** [comment]
Wrapping each tab with if `User.current.alllowed_to?` looks fine to me.

To filter the list by permission, this example should work

``` ruby
- multiple_selects f, :media, Medium, ...
+ multiple_selects f, :media, Medium.authorized(:view_media), ...
```

**Comment by isratrade** [comment]
@thomasmckay, I agree not to include in the PR the filtering of items since it could cause items being unexpected deleted if we don't also submit the hidding items. Ignore my comment above to include `Medium.authorized(:view_media)`.  Looks good to me.

**Comment by isratrade** [comment]
Merged as ae255b3caf953b43b2387bedc78a6a258b2b8b33. thanks @thomasmckay !

**Pull request closed without merging**